dashboard/settings.py

```diff
@@ -241,7 +241,7 @@
     # Gunicorns logging format https://github.com/benoitc/gunicorn/blob/19.x/gunicorn/glogging.py
     'formatters': {
         "generic": {
-            "format": "%(asctime)s [%(process)d] [%(levelname)s] %(message)s",
+            "format": "%(asctime)s [%(levelname)s] [%(filename)s:%(lineno)d] %(message)s",
```

Comment:
This is cool. Are these variables defined by Gunicorn?

Reply:
These are defined by python logging [LogRecord](https://docs.python.org/3/library/logging.html#logrecord-objects)

I think I copied the format from Gunicorn originally because it was better than what we had, but looking at some of these and running this locally now having the file and line number is pretty nice.